install/empty_data.php

```diff
@@ -2102,6 +2102,8 @@
       'autofill_delivery_date'               => 0,
       'autofill_order_date'                  => 0,
       'tickettemplates_id'                   => 1,
+      'changetemplates_id'                   => 1,
```

Comment:
Should that require a database update?

Reply:
This will make the "default" templates enabled by default on fresh installs.
It doesn't impact anything as the default template is empty.

I don't think we want to impact existing GLPI instances with a database update.
They may have modified or deleted this template (or they may be using a different template).

Reply:
If the value is not strictly needed for the templates to work, ok for me

Reply:
The value is for this setting (Root entity):
![image](https://user-images.githubusercontent.com/42734840/96229334-09926700-0f97-11eb-9c90-194e76de117e.png)